network/Callback.py

```diff
@@ -0,0 +1,54 @@
+import cppyy
+
+from network.NetworkEvent import create_network_event
+
+cppyy.add_include_path("/usr/local/include/SopraClient")
+cppyy.include("Callback.hpp")
+
+
+class Callback(cppyy.gbl.libclient.Callback):
+    """
+    Implements a simple callback class which itself calls the controller
+    """
+
+    def __init__(self, ):
```

Comment:
```suggestion
    def __init__(self):
```

muss die init überhaupt definiert werden, wenn man die eh passed?